common/openapi.yml

```diff
@@ -189,6 +189,10 @@ components:
       properties:
         url:
           type: string
+        cloudPlatform:
```

Comment:
From talking before: it'd be great to have descriptions on these fields and it gives a good way of explaining the behavior of `cloudPlatform`